diff --git a/node_modules/electron-is-accelerator/index.js b/node_modules/electron-is-accelerator/index.js
index 219cfad..1fd0454 100644
--- a/node_modules/electron-is-accelerator/index.js
+++ b/node_modules/electron-is-accelerator/index.js
@@ -1,7 +1,7 @@
 "use strict";
 
 const modifiers = /^(Command|Cmd|Control|Ctrl|CommandOrControl|CmdOrCtrl|Alt|Option|AltGr|Shift|Super)$/;
-const keyCodes = /^([0-9A-Z)!@#$%^&*(:+<_>?~{|}";=,\-./`[\\\]']|F1*[1-9]|F10|F2[0-4]|Plus|Space|Tab|Backspace|Delete|Insert|Return|Enter|Up|Down|Left|Right|Home|End|PageUp|PageDown|Escape|Esc|VolumeUp|VolumeDown|VolumeMute|MediaNextTrack|MediaPreviousTrack|MediaStop|MediaPlayPause|PrintScreen)$/;
+const keyCodes = /^([0-9A-Z)!@#$%^&*(:+<_>?~{|}";=,\-./`[\\\]']|F1*[1-9]|F10|F2[0-4]|Plus|Space|Tab|Backspace|Delete|Insert|Return|Enter|Up|Down|Left|Right|Home|End|PageUp|PageDown|Escape|Esc|VolumeUp|VolumeDown|VolumeMute|MediaNextTrack|MediaPreviousTrack|MediaStop|MediaPlayPause|PrintScreen|num0|num1|num2|num3|num4|num5|num6|num7|num8|num9|numdec|numadd|numsub|nummult|numdiv)$/;
 
 module.exports = function (str) {
 	let parts = str.split("+");
